doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,209 @@
+
+    ### Adding command parameters
+
+    An additional parameter can be added to the method, which is then assigned with the value of the `CommandParameter` received from the View. For instance, when using a Button and clicking it, the method will be called with the [`Button.CommandParameter`](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.primitives.buttonbase.commandparameter) value, given that the type of the value matches the method parameter type. Otherwise the command's `CanExecute` will be false thereby disabling the button:
+
+    ```csharp
+    public void DoWork(double param)
+    {
+        ...
+    }
+    ```
+
+    Then:
+
+    ```xml
+    <Slider x:Name="slider" Minimum="1" Maximum="100"/>
+    <Button Command="{Binding DoWork}" CommandParameter="{Binding Value, ElementName=slider}"/>
+    ```
+
+    Accordingly, if the `CommandParameter` type does not match the parameter type of the method, the button will remain disabled:
+
+    ```xml
+    <Button Command="{Binding DoWork}" CommandParameter="A string, not a double" />
+    ```
+
+    Result:
+
+    ![A disabled refresh button](../Assets/DisabledButton.jpg)
+        
+    ### Command generation rules:
+    
+     - Can be either synchronous or asynchronous (i.e. `public void`, `public async ValueTask`, or `public async Task`)
+     - Any parameter that has a type and name (case-insensitive) matching a Feed or a State in this Model will be evaluated when the Command is invoked and its current latest value will be passed in as an argument.  
+        You'll find this feature to be very powerful in invoking commands by combining data from various Feeds in addition to a command parameter received from the View.
+
+        For example:
+
+        ```csharp
+        public IFeed<int> CounterValue => ...
+
+        public void ResetCounter(int counterValue)
+        {
+            ...
+        }
+        ```
+
+        When the command is executed, because the parameter name `counterValue` matches a feed name in the Model, this parameter will be materialized with the actual value from the Feed when this method is called on command execution.  
+        This behavior can be controlled and configured using the [`FeedParameter`](#feedparameter-attribute) and [`ImplicitFeedCommandParameter`](#implicitfeedcommandparameter-attribute) attributes.
+
+     - Can have one `CancellationToken` as its last parameter, but it's not mandatory.
+
+     > [!NOTE]  
+     > Name matching of command parameters to Feeds is case-insensitive.
+
+     > [!TIP]
+     > If you are using [Event Binding](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension#event-binding), you might need to opt out from command generation for those methods. See the next section on how to configure command generation.
+
+     ### Using attributes to control command generation
```

Comment:
Headings within this group still seem inconsistent. 
I won't mention it again, but it's probably worth reviewing for all subsequent headings too.